docs/pbc/all/warehouse-management-system/202311.0/unified-commerce/fulfillment-app-feature-overview.md

```diff
@@ -0,0 +1,112 @@
+{% endinfo_block %}
+
+
+## Warehouse picklists
+
+A *warehouse picklist* is a document available in the Fulfillment App that contains a list of items to be picked and shipped to fulfill and order. It includes the following information:
+* Product name
+* Product image
+* Quantity to be picked
+
+Based on a picklist, the warehouse user gathers and prepares the items for shipping.
+
+The information within will also allow you to quickly restock inventory as required.
+
+![screenshot with a piklist]
+
+### Picklist generation strategies
+
+A picklist generation strategies defines how picklists are generated based on the fulfillment requirements of orders per warehouse. On the project level, each warehouse can have its own strategy.
+
+The default picklist generation strategy is designed to generate picklists by order, where each order line is assigned to a unique picklist that contains all the items needed to fulfill the order.
+
+Also, an order can be split into multiple picklists by shipments. If an order contains items that need to be shipped to multiple locations, the order is split into two picklists. This helps the warehouse user ship items more efficiently.
+
+You can also import warehouse strategies by following the (relevant guide)[https://docs.spryker.com/docs/scos/dev/feature-integration-guides/{{page.version}}/install-the-warehouse-picking-feature.html#import-warehouse-picking-list-strategies].
+
+## The picking process
+
+Picking is semi-automated and follows the steps:
+1. The customer places an order.
+2. A warehouse is assigned to the order.
+3. The picklist is generated.
+4. In the Fulfillment App, a warehouse user assigns the picklist to themselves.
+5. The warehouse user marks the items as picked or not found.
+6. Picking is finished.
+7. Order items states are updated to reflect the picking being finished.
+8. The picked items are shipped.
+
+
+## Push notifications
+
+The Fulfillment App supports push notifications. This lets warehouse users receive new picklists without having to refresh the application.
+
+## Offline mode
+
+To reduce costs and improve flexibility, Fulfillment App support offline mode. Warehouse users can perform picking tasks when regardless of internet connectivity. Any changes performed in offline mode are synchronized when you get online. This is relevant for warehouses in remote areas or big warehouses that do not have full network coverage.
+
+## Current constraints
+
+* Supports only regular products. Doesn't support bundles and configurable products.
+* Doesn't support partial picking of non-splittable order items.
+* Doesn't support the business logic for picking of bundle products. However, if all concrete products of a bundle product are picked, it can be processed by the State Machine.
+* Visualization of configurable bundles in Fulfillment App is impossible, but picking of items in bundles is possible.
```

Comment:
Replace "impossible" with not "available yet" or smth similar cos this functionality can be added later on or on a project level.